index.html

```diff
@@ -0,0 +1,315 @@
+              >See project</a
+            >
+          </div>
+        </div>
+        
+      </div>
+      <div class="about-me">
+        <section class="about-me-section">
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+
+    <footer>
+      <div class="footer-form">
+        <h3>Contact Me</h3>
+        <p>If you have an application you are interested in developing. I'd love to help in it</p>
+      <form>
+        <label for="name">Name:</label>
+        <input type="text" id="name" name="name">
+        <label for="email">Email:</label>
+        <input type="email" id="email" name="email" >
+        <label for="description">Description:</label>
+        <input type="text" id="description" name="description" >
```

Comment:
- [ ] Please replace input tag with textarea tag. Because it provides additional information about that document, which aids in communication.

- [ ] Textarea tag should have a maxlength="500" attribute. This will also avoid overlong text blocks being submitted to the form

- [ ] It should also have the required attribute to activate the validation on submit. This is important so that the form can not be submitted without entering a message.
- [ ] Also remove description and replace with messages as stated in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md).
- [ ] Add placeholder to textarea tag as required in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md).